### PR TITLE
Adds YouTube links to the call format documents

### PR DESCRIPTION
**Pull request by GBKS**
Since we now have a YouTube channel where we upload call recordings, this should be pointed out in the documents for call formats.

@ConorOkus I only added it to the community call document. Let me know if you also want me to update the design review doc.

### CODE REVIEW

File: Community-Calls.md [hunk 1]
```diff
@@ -36,16 +36,11 @@ Some examples:
 
 **What tools do we use?**
 
-- We currently use password-protected Zoom calls
-- Google Meet is lacking the moderation options we want
-- [Jitsi](https://jitsi.org) does not scale to the group size we want
-- BigBlueButton has been proposed, but we have not tested it
+We use [Jitsi](https://jitsi.org), which is open-source, privacy-preserving, and available as web and desktop applications.
```
Comment (johnsBeharry) [inline Community-Calls.md:39]:
```suggestion
We use [Jitsi](https://jitsi.org), which is open-source, doesn't require you to create an account, and is available as web and desktop applications.
```

**Review by johnsBeharry — CHANGES_REQUESTED** [review]
just managing expectations wrt the privacy preserving part. at the end of the day it's a public forum, and a hosted service.

**Comment by ConorOkus** [comment]
Thanks @GBKS . Think it makes sense to add to Design Review as well.

**Review by Bosch-0 — DISMISSED** [review]
ACK from me

**Review by Bosch-0 — APPROVED** [review]
Fresh ACK from me

**Review by johnsBeharry — APPROVED** [review]

**Pull request merged**